src/web/App.mjs

```diff
@@ -617,27 +650,26 @@ class App {
 
 
     /**
-     * Resets the splitter positions to default.
+     * Resets the splitter positions to default for desktop UI.
      */
     resetLayout() {
```

Comment:
Is this method still being used? It appears to have been used in `OutputWaiter.mjs`, but this is no longer the case.

Reply:
Yes, it's used on click of the 'reset-layout' button on desktop UI